app/code/Magento/Cms/Test/Unit/Controller/Adminhtml/Block/SaveTest.php

```diff
@@ -373,6 +373,58 @@ public function testSaveAndClose()
         $this->assertSame($this->resultRedirect, $this->saveController->execute());
     }
 
+    public function testSaveActionWithMarginalSpace()
+    {
+        $postData = [
+            'title' => '"><img src=y onerror=prompt(document.domain)>;',
+            'identifier' => '  unique_title_123',
+            'stores' => ['0'],
+            'is_active' => true,
+            'content' => '"><script>alert("cookie: "+document.cookie)</script>',
```

Comment:
I believe that you can set the `title` to some simple test value and leave the `content` empty since we only want to test the content of `identifier` field in this test, right?

Reply:
@krzksz Yes we only want to test the `identifier` of the block. I will change and commit

Reply:
@krzksz I have modified the code and commit the code changes.